ci_framework/roles/edpm_prepare/tasks/main.yml

```diff
@@ -77,6 +77,15 @@
         cmd: >
           oc wait InstallPlan {{ cifmw_edpm_prepare_wait_installplan_out.stdout }}
           --namespace={{ cifmw_install_yamls_defaults['NAMESPACE'] }} --for=jsonpath='{.status.phase}'=Complete --timeout=20m
+
+    - name: Make sure all Openstack operators are deployed
+      ansible.builtin.shell: |
+        set -o pipefail && oc get csv -l operators.coreos.com/openstack-operator.openstack --no-headers=true | grep -i "succeeded"
```

Comment:
Maybe is cleaner by getting the yaml and asserting based on that yaml fields. Specially if we can get from there errors about the actual failure.

Reply:
:+1:  to @pablintino proposal.

Also, we have to ensure we pass this:
```
environment:
  PATH: "{{ cifmw_path }}"
```
in order to get `oc`.

Reply:
After a quick discussion:
- add a TODO
- merge as-is
- refine in a future iteration.